tools/clang/unittests/HLSL/VerifierTest.cpp

```diff
@@ -199,6 +200,13 @@ TEST_F(VerifierTest, RunShaderMismatch) {
   CheckVerifiesHLSL(L"shader_attribute_no_mismatch.hlsl");
 }
 
+TEST_F(VerifierTest, RunMaxRecordsAttribute) {
+  CheckVerifiesHLSL(L"max_output_records_duplicate.hlsl"); 
```

Comment:
Is it possible to make these tests lit FileCheck test?

Reply:
No, this is targeting the release branch.

Reply:
Why release branch cannot have lit FileCheck test?

Reply:
Because we won't hide taef for release branch?

Reply:
Do we plan to merge the LIT change to preview branch?

Reply:
I think that's a question for @pow2clk (#5478)